python/pyarrow/tests/test_parquet.py

```diff
@@ -442,6 +442,7 @@ def test_parquet_metadata_api():
     # Row group
     rg_meta = meta.row_group(0)
     repr(rg_meta)
+    repr(rg_meta.ColumnChunk(0))
```

Comment:
Can you add some tests to show that you can obtain the expected column statistics from each column in a row group? Checking the min and max for a string (bytes) column, integer, and double should be good enough. The idea of ARROW-480 was to be able to know when you can skip reading row groups based on a comparison predicate (like a value range)